### ISSUE TITLE
prevent ghinja from running when dock isn't open or allow some other method to disable

### ISSUE THREAD

**Issue by psifertex** [Martyx00/ghinja#2]
It would be nice to have ghinja only enabled some of the time. Right now the only way to do that is to totally disable/enable the entire plugin. It would be nice if it either didn't run if the dockwidget was hidden, or if there's problems with that implementation a setting to disable/enable would be nice since plugin disabling requires a restart whereas a setting wouldn't.

**Comment by Martyx00** [comment]
Will be changed in the next release :)

**Comment by Martyx00** [comment]
With release 1.1 the Ghidra decompiler starts only after you open ghinja window and click inside it. Closing :)
